tests/test_vtk.py

```diff
@@ -35,6 +42,40 @@ def test_validation_pluto(fn):
     assert PlutoVtkDataset._is_valid(fn)
 
 
+@pytest.mark.parametrize(("fn", "geometry"), PLUTO_VTK_FILES)
+def test_parse_pluto_metadata(fn, geometry):
+    ds = yt_idefix.load(fn)
+    assert os.path.samefile(ds._definitions_header, fn.parent.joinpath("definitions.h"))
+    assert ds.geometry == geometry
+    if fn == PLUTO_SOD:
+        assert ds.current_time == -1
+    else:
+        assert ds.current_time >= 0
+
+
+def test_pluto_sedov_units():
+    ds = yt_idefix.load(PLUTO_SEDOV)
+    assert_allclose_units(ds.length_unit, unyt_quantity(3.08567758e21, "cm"))
+    assert_allclose_units(ds.velocity_unit, unyt_quantity(1e5, "cm/s"))
+    assert_allclose_units(ds.mass_unit, unyt_quantity(4.91416082e40, "g"))
+    assert_allclose_units(ds.time_unit, unyt_quantity(3.08567758e16, "s"))
+    assert_allclose_units(ds.magnetic_unit, unyt_quantity(4.58462477e-07, "G"))
+
+
+def test_pluto_value_error():
+    error_msg = "Header file definitions2.h couldn't be found. The 'geometry' keyword argument must be specified."
+    with pytest.raises(ValueError, match=error_msg):
+        yt_idefix.load(PLUTO_SEDOV, definitions_header="definitions2.h")
```

Comment:
Actually now that you added the actual message I think the error should be a `FileNotFoundError` rather than `ValueError`

Reply:
make sense!

Reply:
> yt_idefix/data_structures.py:416:25: G004 Logging statement uses f-string

However, f-string is not suggeted in logging statements by flake8. Should I ignore it, or do it like this?

``` python
warn_msg = f"Failed to retrieve time from {log_file}, setting current_time = -1"
mylog.warning(warn_msg)
```